substrate/frame/support/src/traits/tokens/fungibles/lifetime.rs

```diff
@@ -34,6 +34,37 @@ pub trait Create<AccountId>: Inspect<AccountId> {
 	) -> DispatchResult;
 }
 
+/// Trait for resetting the team configuration of an existing fungible asset.
+pub trait ResetTeam<AccountId>: Inspect<AccountId> {
```

Comment:
What about a `fungible` impl?

Reply:
If we abstract from the implementation of `fungible`, we can assume it might need the same trait. If we look into our implementations, we know, we wont need it for balances pallet. 
I would better not introduce a trait that we do not need not, and likely never.